features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -141,10 +141,83 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
        And I check the canonical option for the tag "Faye Valentine is a sweetie"
        And I check the canonical option for the tag "Ed is a sweetie"
        And I press "Wrangle"
-     Then I should see "The following tags were successfully wrangled to Cowboy Bebop: Faye Valentine is a sweetie, Ed is a sweetie Wrangle Tags for Cowboy Bebop"
+     Then I should see "The following tags were successfully wrangled to Cowboy Bebop: Faye Valentine is a sweetie, Ed is a sweetie"
        And the "Faye Valentine is a sweetie" tag should be canonical
        And the "Ed is a sweetie" tag should be canonical
 
+  Scenario: more wrangling....Freeform
```

Comment:
Could you make this a little more descriptive? What specifically are we testing here? What I see happening is we're making sure we can mass wrangle in the fandoms bin, but I can't tell:

- Why this test needs to create freeform tags
- Why this test needs to create works
- Why the test needs to create a media category